packages/beacon-node/src/network/reqresp/utils/dataColumnResponseValidaiton.ts

```diff
@@ -0,0 +1,66 @@
+import {LogData} from "@lodestar/logger";
+import {ForkSeq, KZG_COMMITMENTS_GINDEX} from "@lodestar/params";
+import {RespStatus, ResponseError} from "@lodestar/reqresp";
+import {ColumnIndex, Slot} from "@lodestar/types";
+import {Logger, bytesToInt, toHex} from "@lodestar/utils";
+import {IBeaconChain} from "../../../chain/interface.js";
+import {IBeaconDb} from "../../../db/interface.js";
+
+export async function logDataColumnSidecarUnavailability(opts: {
+  chain: IBeaconChain;
+  db: IBeaconDb;
+  index: ColumnIndex;
+  slot: Slot;
+  blockRoot?: Uint8Array;
+  logData: Record<string, LogData>;
+}): Promise<void> {
+  const {chain, db, index, slot, blockRoot, logData} = opts;
+  let localLogData: LogData = {...logData, slot};
```

Comment:
preference to have slot first in the log
```suggestion
  let localLogData: LogData = {slot, ...logData};
```